tests/Aspire.Azure.AI.OpenAI.Tests/Aspire.Azure.AI.OpenAI.Tests.csproj

```diff
@@ -6,7 +6,13 @@
 
   <ItemGroup>
     <None Include="$(RepoRoot)src\Components\Aspire.Azure.AI.OpenAI\ConfigurationSchema.json" CopyToOutputDirectory="PreserveNewest" />
+  </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.DotNet.RemoteExecutor" />
```

Comment:
Just wondering if moving this into the individual test projects is useful.

Reply:
Having an upstream dependency with references it doesn't use directly but instead has those "for a friend that may need it" isn't a great idea in my experience. 
It complicates the dependency graph and makes much harder to reason about each individual project requirements. With this change we know exactly which project requires RemoteExecutor.